### PR TITLE
Feature redesign estimate project form

### PR DESCRIPTION
**Pull request by marsolekk**

### CODE REVIEW

File: company_website/admin.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class EmployeesAdmin(AdminImageMixin, admin.ModelAdmin):
 
 class EstimateAdmin(admin.ModelAdmin):
```
Comment (kbeker) [inline company_website/admin.py:16]: Commit which deletes these fields is lack of `[M]` on the beginning. We are always adding this on the beginning of the commit message to show that this commit has model changes

File: company_website/templates/estimate_project/estimate_project_form.haml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   {% include "common_partials/section_header.haml" with title="Estimate your project" subtext="We’ll get to back to you within 24 hours with the next steps" %}
 
 .container.estimate-form
-  %form{:method => "post", :id => "formABC", :onsubmit => 'return gtagReportConversion("{{ URL_PREFIX }}://{{ request.get_host }}{{ thank_you_url }}")'}
+  %form{:method => "post", :id => "formABC"}
```
Comment (kbeker) [inline company_website/templates/estimate_project/estimate_project_form.haml:7]: Please change ID of this form to more proper one.

File: company_website/templates/estimate_project/estimate_project_form.haml [hunk 3]
```diff
@@ -4,7 +4,7 @@
   {% include "common_partials/section_header.haml" with title="Estimate your project" subtext="We’ll get to back to you within 24 hours with the next steps" %}
 
 .container.estimate-form
-  %form{:method => "post", :id => "formABC", :onsubmit => 'return gtagReportConversion("{{ URL_PREFIX }}://{{ request.get_host }}{{ thank_you_url }}")'}
```
Comment (kbeker) [inline company_website/templates/estimate_project/estimate_project_form.haml]: We don't need ads conversion here? Is it done on google tag manager?

File: company_website/static/estimate_project/scripts/unifyRequiredInfoStyle.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+descriptionField = document.getElementById("id_idea_description");
+policyCheckboxField = document.getElementById("id_privacy_policy_accepted");
+
+inputFieldName = document.getElementsByClassName("input-field")[0];
+inputFieldEmail = document.getElementsByClassName("input-field")[1];
+descriptionBox = document.getElementsByClassName("describe-idea-box-input")[0];
+
+let borderErrorColor = "#ff0000"
+let borderCorrectColor = "#d7d7d7"
+let borderErrorShadow = "0px 0px 16px #ff0000"
```
Comment (kbeker) [inline company_website/static/estimate_project/scripts/unifyRequiredInfoStyle.js:12]: These could be `const` instead of `let`

**Review by kbeker — APPROVED** [review]
Small things to fix. Once you done them please merge it too

File: company_website/templates/estimate_project/estimate_project_form.haml (hunk 2 above)
Comment (marsolekk) [reply company_website/templates/estimate_project/estimate_project_form.haml:7]: Done

File: company_website/static/estimate_project/scripts/unifyRequiredInfoStyle.js (hunk 4 above)
Comment (marsolekk) [reply company_website/static/estimate_project/scripts/unifyRequiredInfoStyle.js:12]: Done

File: company_website/templates/estimate_project/estimate_project_form.haml (hunk 3 above)
Comment (marsolekk) [reply company_website/templates/estimate_project/estimate_project_form.haml]: Yes, it is done by GTM now. We set up tag for thank you url and we measure conversion that way.

File: company_website/admin.py (hunk 1 above)
Comment (marsolekk) [reply company_website/admin.py:16]: Done

**Pull request merged**